docs/source/get_involved/code_of_conduct.rst

```diff
@@ -28,40 +29,65 @@ conform to the following Code of Conduct.
 1 Introduction
 --------------
 
-sktime is a community-oriented and -led project. We value the
-involvement of everyone in the community. We are committed to creating a
-friendly and respectful place for learning, teaching and contributing.
-All participants in our in-person events and online communications are
-expected to show respect and courtesy to others at all times.
+sktime is a community-oriented and -led project.
+
+We value the involvement of everyone in the community, and strive to be an open
+community. We commit to provide fair and rewarding opportunities for everyone without
+restriction, to participate, learn, and become leaders of the community. We expect from
+members of our community to create such opportunities for others as have been created
+for them.
+
+We are committed to scientific quality and strive to be an open forum on matters
+of technical content and open community governance. All community members are expected
+to adhere to the principle of free speech in academia and science, and, to the best
+of their abilities, contribute to jointly creating
+the best possible science and the best possible open community.
+
+We are committed to creating a friendly and respectful place for learning,
+teaching and contributing. All participants in our in-person events and online
+communications are expected to show respect and courtesy to others at all times.
 
 To make clear what is expected, everyone participating in activities
 associated with the sktime project is required to conform to this Code
 of Conduct. This Code of Conduct applies to all spaces managed by the
 sktime project including, but not limited to, in-person focus groups and
 workshops, and communications online via GitHub.
 
-The lead investigator of the sktime - Dr Franz Király - is responsible
-for enforcing the Code of Conduct. He can be contacted by emailing
-f.kiraly@ucl.ac.uk.
+The Code of Conduct Committee of sktime is responsible for defining and
+enforcing the Code of Conduct, according to Section 4 below.
+
+Incidents can be reported to the Code of Conduct Committee as outlined in Section 3, and
+are handled in accordance with Sections 3 and 4.
+
+The community council of sktime (see current composition on "roles")
+is responsible for ensuring that resolutions of the Code of Conduct committee
+are adhered to, and can be contacted under info@sktime.org.
 
-Reports may be reviewed by other members of the core development team,
-unless there is a conflict of interest, and will be kept confidential.
```

Comment:
I would not remove this bcz its less democratic then, so can you kindly put that in again?

Reply:
I removed it because there was no process, e.g., who decides who may view what.

I agree with your comment - removing the ability for core devs to view the paper trail makes the process less transparent and the CoC commitee less accountable.

So, I will add it back but define the process.

Reply:
added this:

```
4.6 Audit of Investigation Process
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

At any point in time, core developers may request the number of ongoing and concluded
investigations over the last year before the date of the query.
Any group of three core developers may also request access to all investigation reports
in a time period, or reports to cases with specified properties.
If there is no conflict of interest (as determined by the Code of Conduct Committee),
access will be shared to anonymized reports for a period of 2 weeks, and the reports
must not be copied or otherwise proliferated.
```

Reply:
the clause

```
Reports of code of conduct committee case proceedings will be kept confidential.
```

is now in section 4.3 (where it belongs imo)